_sass/addon/commons.scss

```
@@ -34,7 +34,7 @@ body {
     env(safe-area-inset-bottom) env(safe-area-inset-left);
   color: var(--text-color);
   -webkit-font-smoothing: antialiased;
-  font-family: 'Source Sans Pro', 'Microsoft Yahei', sans-serif;
+  font-family: $body-font;
```

Please change to a more descriptive variable name：

```suggestion
  font-family: $font-family-base;
```